bin/cleanup_alta

```diff
@@ -0,0 +1,4 @@
+docker kill daq-faux-satlej
```

Comment:
let's put this in bin/net_clean (no need to have extra scripts that pretty much do the same thing)